c7n/resources/s3.py

```diff
@@ -448,6 +448,18 @@ def __call__(self, b):
         return True
 
 
+@filters.register('static-website-hosting')
+class StaticWebsiteHosting(Filter):
+    """Finds if bucket is hosting a static website."""
+
+    schema = type_schema('static-website-hosting')
+
+    permissions = ('s3:*',)
```

Comment:
does this even need to be a custom filter.. the builtin value filter covers this

```
- Website: not-null
```